src/filters/simple-filter-directive.js

```diff
@@ -183,13 +183,23 @@ angular.module('patternfly.filters').directive('pfSimpleFilter', function () {
         return foundFilter !== undefined;
       };
 
+      $scope.enforceSingleSelect = function (filter) {
+        if (filter.type === 'select') {
+          _.remove($scope.config.appliedFilters, {title: filter.title, type: filter.type});
+        }
+      };
+
       $scope.addFilter = function (field, value) {
         var newFilter = {
           id: field.id,
           title: field.title,
+          type: field.filterType,
           value: value
         };
         if (!$scope.filterExists(newFilter)) {
+
+          $scope.enforceSingleSelect(newFilter);
+
           $scope.config.appliedFilters.push(newFilter);
 
```

Comment:
I wonder if we should modify the existing filter to use the new selection rather than removing the old and adding the new. This would keep the filter in the same position in the list of filters. This is really a modification of the current filter.

Thoughts?